bin/cog-command

```diff
@@ -5,8 +5,15 @@ import os
 
 bundle_name = os.getenv("COG_BUNDLE")
 command_name = os.getenv("COG_COMMAND")
-class_name = command_name.capitalize()
-full_path = "%s.commands.%s" % (bundle_name, command_name)
+
+# `-` cannot be used in class names in python so
+# since command_name is used directly for klass
+# this doesn't break compatibility for existing bundles
+# because if a bundle uses `-` in the command name
+# pycog3 <= 0.1.27 throws exception
+path = command_name.replace("-",".")
+full_path = "%s.commands.%s" % (bundle_name, path)
+class_name = command_name.split("-")[-1].capitalize()
```

Comment:
This is a nice direction. I wonder if we could put some testing around this to help ensure that we're getting the desired results with all variations of project organizations. I'd feel good about merging this if we can prove that it won't break backward compatibility.